public/index.js

```diff
@@ -26,11 +26,48 @@ var clearContents = function () {
 }
 
 /**
- * Generates the random gift page building all necessary elements
+ * Generate the random gift page building all necessary elements
  */
 var generateGiftPage = function () {
   var hello = document.createElement('p');
   var helloText = document.createTextNode('I am the gift page');
   hello.appendChild(helloText);
   main.appendChild(hello);
+  var giftButton = document.createElement('BUTTON');
```

Comment:
we should keep case consistant

Reply:
naming: we also need to distinguish this gift button from another gift button that we plan to have